lib/StripeResource.js

```diff
@@ -134,13 +134,26 @@ StripeResource.prototype = {
       res.on('end', function() {
         var headers = res.headers || {};
 
+        // For convenience, make Request-Id easily accessible on
+        // lastResponse.
+        res.requestId = headers['request-id'];
+
+        var responseEvent = utils.removeEmpty({
+          api_version: req._requestEvent.api_version,
```

Comment:
I'm glad you mentioned this, because Fun Fact:

Stripe sends the headers as all lowercase, so none of these would have worked.